team-builder/src/components/Form.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import TeamMember from "./TeamMembers";
+
+const Form = () => {
+  const [form, setForm] = useState({
+    firstName: "",
+    lastName: "",
+    role: "",
+    email: ""
+  });
+  const [submit, setSubmit] = useState([]);
+  console.log(submit);
+
+  const formChange = e => {
+    setForm({
+      ...form,
+      [e.target.name]: e.target.value
+    });
+  };
+
+  const formSubmit = e => {
+    e.preventDefault();
+    setSubmit(submit.concat(form));
+  };
+
```

Comment:
Good way of adding additional elements to your array. Another way would be to use the spread operator:

```
setSubmit([
    ...submit,
    form
])
```